Makefile

```
@@ -0,0 +1,112 @@
+## live: start all 5 watch processes in parallel.
+.PHONY: live
+live:
+	make -j5 templ/live run/live tailwind/live esbuild/live live/sync_assets
+
+# ==================================================================================== #
+# QUALITY CONTROL
+# ==================================================================================== #
+
+## audit: run quality control checks (static, vulnerabilities, etc)
+.PHONY: audit
+audit: test docs/check
```

checkmake (0.2.2)

[warning] Target body for "audit" exceeds allowed length of 5 (8).

(maxbodylength)